atlasdb-cassandra-integration-tests/src/test/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraGetCandidateCellsForSweepingTest.java

```diff
@@ -38,4 +43,29 @@ protected KeyValueService createKeyValueService() {
                 CassandraContainer.LEADER_CONFIG,
                 Mockito.mock(Logger.class));
     }
+
+    @Test
+    public void returnCandidateIfPossiblyUncommittedTimestamp() {
```

Comment:
Why is this test back?

Reply:
these are the two tests we decided to leave in place during the revert. (this test is in 0.61.1)